test/e2e/node_drain_timeout.go

```diff
@@ -49,7 +49,8 @@ type NodeDrainTimeoutSpecInput struct {
 	// configured and a MachineDeployment resource that has
 	// spec.template.spec.nodeDrainTimeout configured.
 	// If not specified, "node-drain" is used.
-	Flavor *string
+	Flavor              *string
+	ControlPlaneWaiters clusterctl.ControlPlaneWaiters
```

Comment:
Could we move this up above Flavor and the Block comment in all cases?

Reply:
Moved